docs/administrator/deploy/upgrade.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: Upgrading applications
+---
+
+# Upgrading applications
+Upgrading an application allows for a new version of a released project to be deployed to an existing application. Tasks and process instances that are in progress and based on a previous application version can still be completed, however any new ones started will use the new model definitions. 
+
+Some considerations for upgrading applications to be aware of are that:
+
+* An application can be upgraded to a released project version that is lower than the one currently deployed, however the application version will still increment.   
+* Applications can only be upgraded through the deployment service. This means that if an application was deployed via Helm it cannot be upgraded.
+* [UIs](../../modeling/interfaces.md) will be added and removed between versions based on whether they exist in the project. If only the `type` is changed and the `name` remains the same then the `type` will not be updated between versions. 
+* It is not possible to change an application name when upgrading an application. 
+* It is not possible to use a different Docker image for any service when upgrading an application except for its connectors.
+* It is not possible to edit or add any variables to services when upgrading an application, except for its connectors. 
+
+## Versioning logic
+The version of an application is incremental and independent of the released project version as shown in the following table: 
+
+| Released project version | Application version | 
+| ------------------------ | ------------------- | 
+| 1 | 1 | 
+| 2 | 2 | 
+| 4 | 3 | 
+| 5 | 4 | 
+| 3 | 5 |
+| 10 | 6 | 
+
+## Upgrade steps 
+Use the following steps to upgrade an application in the Administrator Application: 
+
+1. Sign into the Administrator Application. 
+2. Select the **Upgrade** action against an application in the **Application Instances** tab to upgrade it. 
+3. The application name can't be changed and the released project version to upgrade to must be greater than the one currently deployed.
```

Comment:
no this point is not valid anymore, above you have already explained correctly

Reply:
Ah, yes that's true. I've removed this line too.